sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -570,6 +570,50 @@ In [Resource server visiting other resource server] scenario(For better descript
         return "Response from WebApiB.";
     }
     ```  
+
+#### Support setting redirect-uri-template.
+
+This starter supports Reset Redirection Endpoint(authorization server to return responses containing authorization credentials to the client via the resource owner user-agent). Developers can customize the redirect-uri.
```

Comment:
Delete `This starter supports Reset Redirection Endpoint(authorization server to return responses containing authorization credentials to the client via the resource owner user-agent).`,

Just keep `Developers can customize the redirect-uri.`